docs/howto.rst

```diff
@@ -0,0 +1,112 @@
+*************
+How-to guides
+*************
+
+If you have a specific goal but are already familiar with Rockcraft, our How-to guides have more in-depth detail than our tutorials 
+and can be applied to a broader set of applications. 
+
+They'll help you achieve an end result but may require you to understand and adapt the steps to fit your specific requirements.
+
+
+Getting started
+---------------
+
+Rockcraft is **the tool** for building Ubuntu-based and production-grade OCI images, aka ROCKs!
+
+Rockcraft is distributed as a snap. For packing new ROCKs, it makes use of "providers" to execute 
+all the steps involved in the ROCK's build process. At the moment, the supported providers are LXD and Multipass.
+
+Requirements
+............
+
+Before installing the Rockcraft snap, make sure you have the necessary tools and environment to 
+install and run Rockcraft.
+
+First things first, if you are running Ubuntu, Snap is already installed and ready to go:
+
+.. code-block:: sh
+
+    $ snap --version
+    snap    2.57.1
+    snapd   2.57.1
+    series  16
+    ubuntu  22.04
+    kernel  5.17.0-1016-oem
+
+If this is not the case, then please check https://snapcraft.io/docs/installing-snap-on-ubuntu.
+
+For what concerns providers, LXD is the default one for Rockcraft, so start by checking if it is available and enabled:
+
+.. code-block:: sh
+
+  $ lxd --version
+  5.5
+  $ systemctlstatus snap.lxd.daemon.service
+  ● snap.lxd.daemon.service - Service for snap application lxd.daemon
+      Loaded: loaded (/etc/systemd/system/snap.lxd.daemon.service; static)
+      Active: active (running) since Wed 2022-09-07 16:02:29 CEST; 6 days ago
+      ...
+
+If LXD is not installed, then run:
+
+.. code-block:: sh
+
+  $ snap install lxd
+
+And if LXD is not running, try starting it via:
+
+.. code-block:: sh
+
+  $ lxd init
+
+
+May you find any problems with LXD, please check https://ubuntu.com/lxd.
+
+It is also useful to ensure also Multipass is installed and available:
+
+.. code-block:: sh
+
+    $ multipass version
```

Comment:
There are four (I think) spaces of indentation here, as opposed to the two spaces in the previous command samples. Does that make a difference in the output? 
(yes I agree it's a weird thing to notice)

Reply:
:p fixed and added `sphinx-lint` to the CI